eclass/qt6-build.eclass

```diff
@@ -0,0 +1,161 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+# @ECLASS: qt6-build.eclass
```

Comment:
Shouldn't qmake-utils.eclass be here too for the qt6_get_bindir stuff? (note meant to sync it with ::gentoo back in https://github.com/gentoo/qt/pull/253/commits/67fea4472f136c2cc4bde966c10df97b83d13388 but tossed that PR).

On a related note wrt bindir, qt5 is now installing /usr/bin/qmake5, it may make sense to have qmake6 there too. Guess could potentially saves from having to use qt*_get_bindir at times.

Also may want to bump qt6-build.eclass' copyright year, albeit I imagine haven't technically touched it since 2021 :eyes:

Reply:
> On a related note wrt bindir, qt5 is now installing /usr/bin/qmake5 (symlink), it may make sense to have qmake6 there too. Guess could potentially save from having to use qt*_get_bindir at times.

I could add that my PyQt6{,_WebEngine} ebuilds actually wouldn't need qmake-utils updates with that, could drop the qt6_get_bindir and call qmake6 from PATH :eyes:

Reply:
Mmmm, maybe yeah.  Haven't really looked at the eclasses in a while myself.  I'll leave this un-done for the time being and push the changes I've made now.

If you think it's a good idea, it's probably a good idea!